.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
add `.vscode` to `.gitignore`  file to remove this file

Reply:
Alright, I rushed to push cos of light, I will see to it now, though I am new to ejs files and others, I am honestly a new bie to javascript and co